TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/BotActivity.java

```diff
@@ -177,8 +177,15 @@ public void onServiceDisconnected(ComponentName componentName) {
 
     @Override
     public void onConnect(String message) {
-        debugView.showPermanently(getResources().getString(R.string.connected));
+        debugView.showPermanently(getString(R.string.connected));
         switchableView.setDisplayedChild(1);
+        presenter.startListeningForDirection();
```

Comment:
Wondering if I shouldn't drop this from the activity and just have the presenter automatically start listening to moves on connection?